.github/workflows/tests.yml

```
@@ -88,7 +91,7 @@ jobs:
         with:
           file: ./coverage.xml
 
-  math:
+  mixed-interfaces-tests:

```

Minor: how about adjusting it to the name of the marker?
```suggestion
  all-interfaces-tests:
```

---

👍